### PR TITLE
added model book, and repo with service.

### PR DESCRIPTION
**Pull request by opopovych**

### CODE REVIEW

File: src/main/java/mate/academy/model/Book.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+@Entity
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Nonnull
```
Comment (Rommelua) [inline src/main/java/mate/academy/model/Book.java]: @Column(nullable = false)

**Review by Rommelua — CHANGES_REQUESTED** [review]

File: mvnw [hunk 2]
```diff
@@ -0,0 +1,308 @@
+#!/bin/sh
```
Comment (koval-kn) [inline mvnw:1]: in future, don't include this file to the git

File: mvnw.cmd [hunk 3]
```diff
@@ -0,0 +1,205 @@
+@REM ----------------------------------------------------------------------------
```
Comment (koval-kn) [inline mvnw.cmd:1]: dont include this too

File: .mvn/wrapper/maven-wrapper.properties [hunk 4]
```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.5/apache-maven-3.9.5-bin.zip
```
Comment (koval-kn) [inline .mvn/wrapper/maven-wrapper.properties:1]: don't include this file too, as like as maven-wrapper.jar

File: src/main/java/mate/academy/OnlineBookStoreApplication.java [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        return args -> {
+            Book book = new Book();
+            book.setTitle("Java");
+            book.setAuthor("Hortsman");
+            book.setIsbn("001");
+            book.setPrice(BigDecimal.valueOf(300));
+            book.setCoverImage("image");
+            bookService.save(book);
+            System.out.println(bookService.findAll());
+
```
Comment (koval-kn) [inline src/main/java/mate/academy/OnlineBookStoreApplication.java:32]: redundant blank line

File: src/main/java/mate/academy/OnlineBookStoreApplication.java [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        };
+    }
+
```
Comment (koval-kn) [inline src/main/java/mate/academy/OnlineBookStoreApplication.java:35]: redundant blank line

**Review by koval-kn — APPROVED** [review]
see comments

**Pull request merged**